src/table.tsx

```diff
@@ -178,7 +178,9 @@ function formatTextWithMargins({
     }
   }
 
-  const text = cliTruncate(valueWithNoZeroWidthChars, spaceForText, {position: determineTruncatePosition(overflow)})
+  const text = cliTruncate(valueWithNoZeroWidthChars.replaceAll('\n', ' '), spaceForText, {
+    position: determineTruncatePosition(overflow),
+  })
```

Comment:
It would be good to create a simple test for this newline replacement so we don't have a regression later.

Also, unrelated: this uncovered an issue with `align-center` where the first line is not centered.

```
Wrap (aligned center)
┌───────┬─────────┬─────┬───────────────────────────────────────────────────────────────────────────┐
│  Id   │  Name   │ Age │                                Description                                │
├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
│ 36329 │  Alice  │ 20  │  Lorem ipsum dolor sit amet, consectetur                                  │
│       │         │     │                                   adipi                                   │
│       │         │     │  scing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna   │
│       │         │     │                                  aliqua.                                  │
├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
│ 49032 │   Bob   │ 21  │  Lorem ipsum dolor sit amet, consectetur                                  │
│       │         │     │                                   adipi                                   │
│       │         │     │  scing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna   │
│       │         │     │                                  aliqua.                                  │
├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
│ 51786 │ Charlie │ 22  │  Lorem ipsum dolor sit amet, consectetur                                  │
│       │         │     │                                   adipi                                   │
│       │         │     │  scing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna   │
│       │         │     │                                  aliqua.                                  │
└───────┴─────────┴─────┴───────────────────────────────────────────────────────────────────────────┘
```